client/src/app/site/services/openslides.service.ts

```diff
@@ -25,19 +25,15 @@ export class OpenSlidesService {
      * {@method afterLoginBootup}. If not, redirect the user to the login page.
      */
     public async bootup(): Promise<void> {
-        const online = await this.authService.doWhoAmIRequest();
+        const online = await this.authService.checkOnline();
```

Comment:
If I understand correctly this now always does a request to a health endpoint on bootup. I think that would be unnecessary.